spec/unit/prepare.spec.js

```diff
@@ -783,6 +783,20 @@ describe('prepare', () => {
             Api = rewire('../../bin/templates/cordova/Api');
             prepare = rewire('../../bin/templates/cordova/lib/prepare');
 
+            prepare.__set__('require.resolve', (file) => {
+                if (file === 'cordova-android/framework/defaults.json') {
+                    return path.resolve('./framework/defaults.json');
+                }
+                return path.resolve(file);
+            });
+
+            const defaults = require('../../framework/defaults.json');
+
+            prepare.__set__('fs', {
+                readJSONSync: () => defaults,
```

Comment:
All reads will return the defaults. Is this desired? Moreover, the same object instance will be returned to all tests, making them prone to influence each other.

Reply:
We can remove the test and figure something else out in the future.

We need to push forward with the release since we have less then a month before Google's Playstore new hard requirements. We should get something released as soon as possible to get user feedback on bugs.